minkindr/test/test-transformation-2d.cc

```diff
@@ -0,0 +1,123 @@
+#include <cmath>
+
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+#include <gtest/gtest.h>
+
+#include "kindr/minimal/transform-2d.h"
+
+#ifndef TEST
+#define TEST(a, b) int Test_##a##_##b()
+#endif
+
+#define __INTERNAL_GTEST_NEAR_EIGEN(PREDICATE, matrix_A, matrix_B, precision) \
```

Comment:
I think double underscores should be avoided. [They seem to be reserved](https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier/228797#228797), but not sure if that also applies to macros.